src/core/Buttons/DropdownButton/DropdownButton.test.tsx

```diff
@@ -10,7 +10,7 @@ import { MenuItem } from '../../Menu';
 import SvgCaretDownSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretDownSmall';
 import SvgCaretUpSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretUpSmall';
 
-function renderComponent(...props: Partial<DropdownButtonProps>[]) {
+function renderComponent(props?: Partial<DropdownButtonProps>) {
```

Comment:
The previous syntax was not working, causing the test to fail with this error
![image](https://user-images.githubusercontent.com/9084735/115460074-cfc84480-a1f5-11eb-9403-9b257fb7d1ce.png)

Reply:
I went through all instances of "Partial" and found the same in SplitButton so I fixed there too

Reply:
What, so it was not working all the time? How tests were passing then..

Reply:
Oh ok.. We were not passing anything, so it was not complaining... Interesting